cmd/keymaster/main_test.go

```diff
@@ -335,7 +335,7 @@ func TestGetParseURLEnvVariable(t *testing.T) {
 //  gopass checks
 func TestPipe(t *testing.T) {
 	_, err := pipeToStdin("password\n")
-	_, password, err := getUserInfoAndCreds()
+	password, err := getUserCreds("userame")
```

Comment:
"userame" You meant "username"?

Does this change mean that the prompt will be "Password for useame: "?